CONTRIBUTING.md

```diff
@@ -35,3 +35,41 @@ Examples:
 ./run-tests.sh 5.4 ./link
 ```
 
+# Contributor ladder
+
+If you'd like to contribute to the library more regularly, one of the
+[maintainers][ebpf-lib-maintainers] can add you to the appropriate team or mark
+you as a code owner.
+Just create an issue in the repository.
+
+## [ebpf-lib-contributors]
+
+Contributors support the development of the library by writing code, diagnosing
+bugs, triaging issues, answering questions, etc.
+
+* Can request reviews
+* Can triage issues and pull requests
+* May be asked to review certain parts of code
+* May be asked to help with certain issues
+
+## Code owners
```

Comment:
We currently don't have a CODEOWNERS file, should we add an initial one with this PR?

Reply:
I'll add

```
* @ebpf-lib-maintainers
```

as a start. If I understand correctly that would always request a review from either Timo or me, which is essentially what happens already.

There is another [bit here](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners):

> The people you choose as code owners must have write permissions for the repository.

The implications of write permission are wider than I'm proposing: https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/repository-roles-for-an-organization Specifically a CODEOWNER can merge PRs (subject to branch protection rules) and create new releases. I'd prefer to restrict cutting a release to maintainers, maybe we can use [tag protection rules](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/configuring-tag-protection-rules) to prevent creating `v*` tags?

Otherwise we could "emulate" code owners manually by asking individuals for review and putting responsibilites into a `.md`.